package.json

```diff
@@ -120,10 +120,13 @@
   "devDependencies": {
```

Comment:
`npm ci` is failing, because that does not automatically install peerDependencies